mitodl/mitxonline
# Added Wagtail and initial model definitions

#### Pre-Flight checklist


#### What are the relevant tickets?
N/A

#### What's this PR do?
Adds Wagtail, defines some barebones initial models, and a template tag to allow us to serve Wagtail images without using the Wagtail image view

#### How should this be manually tested?
- Run the `configure_wagtail` management command
- Go to `/cms`, upload an image or two, set an image for the `hero` in the home page, and add a product page as a child of the home page. Make sure you can see the hero image on the home page and that both pages load (they're intentionally _very_ barebones)
